script/codecov.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# https://github.com/codecov/example-go#caveat-multiple-files
+
+cd `dirname $0`/.. || exit 1
+echo "" > coverage.txt || exit 1
+for d in $(go list ./... | grep -v vendor); do
+  go test -race -coverprofile=profile.out -covermode=atomic $d || exit 1
```

Comment:
Do not use `-v` option?

Reply:
Thank you for your review.
I have added `-v` option.
https://github.com/line/line-bot-sdk-go/pull/142/commits/258ad236b899b8b07131e5ac66b5be789aeb3040

Reply:
Thanks! LGTM 🙆‍♂️